proxy/envoy/config.go

```diff
@@ -106,11 +106,11 @@ func Generate(context *ProxyContext) *Config {
 			Clusters: clusters,
 			SDS: &SDS{
 				Cluster:        buildDiscoveryCluster(mesh.DiscoveryAddress, "sds"),
-				RefreshDelayMs: 1000,
+				RefreshDelayMs: (int)(mesh.DiscoveryRefreshDelay.Nanoseconds() / 1e6),
```

Comment:
I based this on Russ Cox's recommendation [here](https://github.com/golang/go/issues/5491#issuecomment-66079585). 

That results in mismatched types because `time.Millisecond` is of type `time.Duration` and golang doesn't do implicit type conversion (see https://play.golang.org/p/Qq1hSq6AVm)